spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -976,6 +974,115 @@ public Message<O> process(Message<I> msg) {
 The implementation is reponsible for enforcing the acknowledgement strategy defined by the user when the `@Acknowledgement` policy is used. 
 If the annotation is not used, the default policy must be enforced.
 
+=== Connector
+
+Reactive Messaging connects matching `@Incoming` and `@Outgoing` running inside the same application.
+Besides, it maps specific _channels_ to _external_ technologies such as Apache Kafka, MQTT, Web Sockets, or JMS. 
+It means that Reactive Messaging can receive messages from virtually any messaging technology and dispatch messages to any messaging technology.
+The component responsible for this role is called _connector_. 
```

Comment:
Should be
"The components responsible for this role are called _connectors_."
or
"This bridging to an external messaging technology is done using a  reactive messaging _connector_."